doc/CDSL.md

```diff
@@ -1 +1,22 @@
 # CDSL
+CDSL is Component Definition Language used to create and define component in Robocomp. With CDSL you specify how the component will communicate with the world. 
+
+A basic cdsl file can be generated as given below. Go to a folder and type the following commands to make a directory and generate a component :
+https://robocomp.github.io/web/gsoc/2017/love/post5
+
+So after this we can compile the component using the following command :
+robocompdsl myComponent.cdsl .
+
+So this generates two new folders :
+
+1. etc
+2. src
+
+The src folder contains a folder and two files:
+1. UI (folder)
+2. component.js
```

Comment:
.js is not a good example for cdsl

Reply:
Ok...I take this point in my bucket of knowledge.
Thanks!!